src/olympia/reviewers/forms.py

```diff
@@ -250,13 +251,15 @@ def create_option(
         # label_from_instance() on WidgetRenderedModelMultipleChoiceField returns the
         # full object, not a label, this is what makes this work.
         obj = label
-        forwarded_notes = [
-            *obj.queue_moves.values_list('notes', flat=True),
-            *obj.decisions.filter(action=DECISION_ACTIONS.AMO_REQUEUE).values_list(
-                'private_notes', flat=True
-            ),
-        ]
         is_appeal = obj.is_appeal
+        queue_moves = list(obj.queue_moves.order_by('-created'))
+        requeued_decisions = list(
+            obj.decisions.filter(action=DECISION_ACTIONS.AMO_REQUEUE).order_by(
+                '-created'
+            )
+        )
+        forwarded = queue_moves[0].created if queue_moves else None
+        requeued = requeued_decisions[0].created if requeued_decisions else None
```

Comment:
Nit, but imo we don't need both forwarded and requeued datetimes - just whatever is the most recent.

Reply:
I wanted to split Forwarded from Requeued text to make things more explicit, potentially displaying both, so adding extra logic to hide the date on one of them in that case felt a bit too much. I'll keep it that way until someone complains...